Packs/CortexResponseAndRemediation/Playbooks/silent-playbook-A_user_executed_multiple_LDAP_enumeration_queries_Test.yml

```diff
@@ -315,9 +309,7 @@ tasks:
     skipunavailable: false
     task:
       brand: ''
-      description: Checks if investigation and remediation can be done based on pre-conditions
-        signifying high probability of a true positive alert and inherently malicious
-        behavior.
+      description: Checks if investigation and remediation can be done based on pre-conditions signifying high probability of a true positive alert and inherently malicious behavior.
```

Comment:
Checks whether investigation and remediation can be done based on preconditions indicating high probability of a true positive alert and inherently malicious behavior.